exonum-java-binding-core/rust/ejb-app/TUTORIAL.md

```diff
@@ -54,39 +47,41 @@ EJB App configuration is pretty similar to configuration of any other Exonum ser
 with a few additional parameters.
 
 #### Generate Template Config
+Use `--ejb-module-name` for fully qualified name of your `ServiceModule`.
 
 ```$sh
-$ ejb-app generate-template testnet/common.toml
+$ ejb-app generate-template testnet/common.toml \
+    --validators-count=1
+    --ejb-module-name 'com.company.project.ServiceModule'
 ```
 
 #### Generate Node Private and Public Configs
 
-- `--ejb-classpath` for a classpath of your service.
-- `--ejb-libpath` for a path to Java bindings native libraries.
-
 ```$sh
 $ ejb-app generate-config testnet/common.toml testnet/pub.toml testnet/sec.toml \
-    --ejb-classpath $EJB_CLASSPATH \
-    --ejb-libpath $EJB_LIBPATH \
     --peer-address 127.0.0.1:5400
 ```
 
 #### Finalize Configuration
 
-There are two specific parameters here:
-- `--ejb-module-name` for fully qualified name of your `ServiceModule`.
-- `--ejb-port` for port that your service will use for communication.
-  Java Binding does not use Exonum Core ports directly.
+Use `--ejb-classpath` for a [classpath](#CLASSPATH) of your service.
 
 ```$sh
 $ ejb-app finalize testnet/sec.toml testnet/node.toml \
-    --ejb-module-name 'com.<company-name>.<project-name>.ServiceModule' \
-    --ejb-port 6000 \
+    --ejb-service-classpath $CLASSPATH \
     --public-configs testnet/pub.toml
 ```
 
 ### Step 3. Run Configured Node
+There are two specific parameters here:
+- `--ejb-log-config-path` for path to `log4j` configuration file.
+  Default config provided with EJB App prints to STDOUT.
+- `--ejb-port` for port that your service will use for communication.
+  Java Binding does not use Exonum Core ports directly.
```

Comment:
the Exonum Core port or the Exonum Core API port? — it is a replacement for a single port